bam_markdup.c

```diff
@@ -948,7 +948,6 @@ static int bam_mark_duplicates(md_param_t *param) {
                 "EXAMINED: %d\n"
                 "PAIRED: %d\n"
                 "SINGLE: %d\n"
-                "DULPICATE PAIR: %d\n"
```

Comment:
it looks like the line has been removed, but not added back with the correct spelling?

Reply:
Oh, for crying out loud!  I even had someone check it for me.  Thank you.